west.yml

```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 4579f028e1e6139786491e352c50bc9528b09c7d
+      revision: pull/1264/head
```

Comment:
Please update SHA: `c9d01d05ce83a8cec0fd60bf8e2ba27c1534d08f`

Reply:
Thanks.
Updated the manifes file.